build/x86_64_centos6/Dockerfile

```diff
@@ -15,15 +17,12 @@ RUN yum update -y && yum install -y \
   rpm-sign \
   autoconf \
   expect \
-  centos-release-SCL
-
-ENV PATH ${PATH}:/usr/local/bin
-
-RUN rpm -ivh http://repoforge.mirror.constant.com/redhat/el6/en/x86_64/rpmforge/RPMS/rpmforge-release-0.5.3-1.el6.rf.x86_64.rpm \
+  centos-release-SCL \
+  && rpm -ivh http://ftp.tu-chemnitz.de/pub/linux/dag/redhat/el6/en/x86_64/rpmforge/RPMS/rpmforge-release-0.5.3-1.el6.rf.x86_64.rpm \
   && yum --enablerepo=rpmforge-extras install -y git \
   && gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 \
   && git config --global user.email "stackdriver-github-reader@google.com" \
-  && git config --global user.name "stackdriverreader"
+  && git config --global user.name "stackdriverreader" \
+  && curl -sSL https://get.rvm.io | /bin/bash -s stable
```

Comment:
Careful! This pipe will probably not do what you expect. Let's put the pipe in a subshell...

Reply:
Done.